intellij-toml/src/test/kotlin/org/toml/ide/formatter/TomlFormatterTest.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.toml.ide.formatter
+
+class TomlFormatterTest : TomlFormatterTestBase() {
+    fun `test spacing`() = doTest("""
+[  config . subconfig ]
+    list  =   [  1,2,     3   ]
+key1 . key2='value'
+object={a=1,b=2}
+    """, """
+[config.subconfig]
+list = [1, 2, 3]
+key1.key2 = 'value'
+object = { a = 1, b = 2 }
+    """)
```

Comment:
```suggestion
    fun `test spacing`() = doTest("""
        [  config . subconfig ]
            list  =   [  1,2,     3   ]
        key1 . key2='value'
        object={a=1,b=2}
    """, """
        [config.subconfig]
        list = [1, 2, 3]
        key1.key2 = 'value'
        object = { a = 1, b = 2 }
    """)
```

I know that test will not pass without additional changes. Suggest using `trimIndent` for expected/actual texts in some base method. For example, in `checkByText`